templates/server/src/main/resources/archetype-resources/server/pom.xml

```diff
@@ -29,10 +29,6 @@
       <version>${project.version}</version>
     </dependency>
     #end
-    <dependency>
-      <groupId>org.flywaydb</groupId>
-      <artifactId>flyway-core</artifactId>
```

Comment:
was that redundant? Already in core I guess. Fine then. Thanks for cleanup.

Reply:
https://github.com/oasp/oasp4j/issues/664

Reply:
jep referenced in core and was temporary added as described in #664. All perfect.